packages/core/src/index.ts

```diff
@@ -200,77 +211,57 @@ class Hopr extends EventEmitter {
     await this.connector.start()
     verbose('Started HoprEthereum. Waiting for indexer to find connected nodes.')
 
+    // Fetch previous announcements from database
     const initialNodes = await this.connector.waitForPublicNodes()
-    verbose('Initial public nodes', initialNodes)
+
+    // Fetch all nodes that will announces themselves during startup
     const recentlyAnnouncedNodes: PeerStoreAddress[] = []
     const pushToRecentlyAnnouncedNodes = (peer: PeerStoreAddress) => recentlyAnnouncedNodes.push(peer)
     this.connector.on('peer', pushToRecentlyAnnouncedNodes)
 
-    const libp2p = await createLibp2pInstance(this.id, this.options, initialNodes, this.publicNodesEmitter)
```

Comment:
remove legacy startup behavior and prevent from awaiting things before they are ready